web_external/js/views/body/SessionView.js

```diff
@@ -93,40 +117,46 @@ minerva.views.SessionView = minerva.View.extend({
             this._disableSave();
         });
 
-        this.dataPanel = new minerva.views.DataPanel({
-            session: this.model,
-            collection: this.datasetsCollection,
-            parentView: this
-        });
-
-        this.mapPanel = new minerva.views.MapPanel({
-            session: this.model,
-            collection: this.datasetsCollection,
-            parentView: this
-        });
-
-        this.layersPanel = new minerva.views.LayersPanel({
-            collection: this.datasetsCollection,
-            parentView: this
-        });
-
-        this.jobsPanel = new minerva.views.JobsPanel({
-            parentView: this
-        });
-
-        this.analysisPanel = new minerva.views.AnalysisPanel({
-            parentView: this,
-            collection: this.analysisCollection,
-            datasetCollection: this.datasetCollection,
-            sourceCollection: this.sourceCollection
-        });
-
-        this.sourcePanel = new minerva.views.SourcePanel({
-            session: this.model,
-            sourceCollection: this.sourceCollection,
-            datasetCollection: this.datasetsCollection,
-            parentView: this
-        });
+        this.layout = {
```

Comment:
From initial message:
> Storing the layout on the session view is sloppy, it should probably be on the model - though it seems like that will be part of a larger refactor in #225

I could do it now, but I feel like a lot of the session is going to be ripped apart shortly.